quickstarts/ansible-tower/config.yml

```diff
@@ -0,0 +1,67 @@
+
+# Name of the quickstart
+slug: ansible-tower
+
+# Title of the quickstart
+title: Ansible Tower
+
+# Summary of the quickstart
+summary: |
+  The Ansible Tower integration will enable you to monitor and analyze your Ansible Tower infrastructure.
+
+# Description of the quickstart
+description: |
+  ## Comprehensive monitoring quickstart for Ansible Tower
+
+  Monitor your Ansible Tower with New Relic's Ansible Tower monitoring system in real-time, correlate transactions traversing across your application environment, and diagnose while running in a live development environment. 
+
+  ### Why monitor Ansible Tower?
+
+  An ideal Ansible Tower Performance Monitor must offer comprehensive and actionable information that software developers need to troubleshoot an application successfully. Some key components are:
+
+  - Track the efficiency of business transactions starting using the Ansible Tower Server
+  - Use deep diagnostics to analyze and fix issues with application performance.
+  - Drill down to the Ansible Tower and the associated application components.
+  - Detects faults to reduce application downtime.
+  - Compare the performance of your Ansible tower server.
+
+  ### What’s included in this quickstart?
+
+  New Relic's Ansible monitoring quickstart boasts instant full-stack observability out-of-the-box:
+
+  - High value alerts for your Ansible Towers (errors,usage metrics and Awx)
+  - Easy to use performance monitoring dashboards (most successful transactions, average transaction duration today compared with that for the previous week, Awx organization comparisons, and more)
+  - Custom alerts that proactively inform developers about the status of their Ansible Tower servers.
+
+# Support level
+level: New Relic
+
+# Design
+icon: logo.png
+
+# Authors of the quickstart
+authors:
+  - New Relic
+
+# Here are the details of the "Data Sources" section of details page
+documentation:
+  - name: Ansible Tower Setup
+    description: |
+      Ansible Tower observability quickstart contains 1 data source. This is how you'll get your data into New Relic.
+    url: https://docs.newrelic.com/
```

Comment:
Looks like we just need to get this url and the one in the install plan up to date and we should be good to go!